src/main/js/main_html5.js

```diff
@@ -494,6 +497,9 @@ require("../../../html5-common/js/utils/environment.js");
           var mode = (!!params && params.mode) || 'showing';
 
           $(_video).append("<track class='" + TRACK_CLASS + "' kind='subtitles' label='" + label + "' src='" + src + "' srclang='" + language + "' default>");
+          if (_video.textTracks && _video.textTracks[0]) {
+            _video.textTracks[0].oncuechange = onClosedCaptionCueChange;
```

Comment:
Correct. Each textTrack will have this callback for its cue change but it will only get fired on the active one. Also, in the onClosedCaptionCueChange function, it makes sure to only extract text from active cues.